### PR TITLE
Improve Windows file permissions handling

### PR DESCRIPTION
**Pull request by stooke**
This PR updates the file permissions in the checked out OpenJDK repo.  

The Cygwin64 git does not create files with proper Windows file permissions when it clones a remote repository.
This PR reworks the code that fixes those permissions.

After this PR is merged, the upstream OpenJDK8u builds run without issue.

### CODE REVIEW

File: build8.cmd [hunk 1]
```diff
@@ -426,6 +426,7 @@
 	set m2=
 	set m3=
 	set m4=
+	set m5=
```
Comment (jerboaa) [inline build8.cmd]: I suppose this is an unrelated change?

File: build8.cmd [hunk 2]
```diff
@@ -442,6 +443,9 @@
 		set m2=tools/bootjdk7 tools/cmake tools/cygwin_jdk11 tools/make tools/zip
 	)
 	set m3=tools/toolchain/directx tools/toolchain/msvcr100 tools/toolchain/sdk71 tools/toolchain/vs2010e
+	if defined WITH_JTREG (
+		set m5=tools/jtreg42_653
+	)
```
Comment (jerboaa) [inline build8.cmd]: Same here?

**Review by jerboaa — COMMENTED** [review]
Could we reduce unrelated changes, please?

File: build8.cmd (hunk 1 above)
Comment (stooke) [reply build8.cmd]: Gone.

**Review by jerboaa — APPROVED** [review]
OK with me. Thanks!
